test/example-expiration-template

```diff
@@ -0,0 +1,8 @@
+Hello,
+
+Your Lets Encrypt SSL certificate for common name {{.CommonName}} (and DNSNames {{.DNSNames}}) is
```

Comment:
I've gotten feedback from others that they want this repo to be generic with respect to Let's Encrypt. So maybe instead of "Let's Encrypt" let's say "happy hacker fake CA." :-)